README.md

```diff
@@ -38,25 +55,25 @@ This will then set the appropriate server endpoint in `environment.ts`
 
 Run project:
 
-```bash
+```
 yarn start
 ```
 
 Build project
 
-```bash
+```
 yarn run build
 ```
 
 Run storybook server
 
-```bash
+```
 yarn run storybook
 ```
 
 Run ESLint
```

Comment:
Should we put an additional note in here telling people to run this command prior to submitting a PR?

Reply:
yes good idea